stone/target/python_rsrc/stone_serializers.py

```diff
@@ -226,23 +226,27 @@ def encode_primitive(self, validator, value):
         if validator in self.alias_validators:
             self.alias_validators[validator](value)
 
-        if isinstance(validator, bv.Void):
-            return None
-        elif isinstance(validator, bv.Timestamp):
-            return value.strftime(validator.format)
-        elif isinstance(validator, bv.Bytes):
-            if self.for_msgpack:
-                return value
+        try:
+            if isinstance(validator, bv.Void):
+                return None
+            elif isinstance(validator, bv.Timestamp):
+                return value.strftime(validator.format)
+            elif isinstance(validator, bv.Bytes):
+                if self.for_msgpack:
+                    return value
+                else:
+                    return base64.b64encode(value).decode('ascii')
+            elif isinstance(validator, bv.Integer) \
+                        and isinstance(value, bool):
+                # bool is sub-class of int so it passes Integer validation,
+                # but we want the bool to be encoded as ``0`` or ``1``, rather
+                # than ``False`` or ``True``, respectively
+                return int(value)
             else:
-                return base64.b64encode(value).decode('ascii')
-        elif isinstance(validator, bv.Integer) \
-                    and isinstance(value, bool):
-            # bool is sub-class of int so it passes Integer validation,
-            # but we want the bool to be encoded as ``0`` or ``1``, rather
-            # than ``False`` or ``True``, respectively
-            return int(value)
-        else:
-            return value
+                return value
+        except Exception as exc:
```

Comment:
What exceptions do you expect here?  Would it be enough to limit the `try` to the `b64encode` call?